### PR TITLE
Gh 2402 integration test support for multiple rest services

### PR DESCRIPTION
**Pull request by m29827**

### CODE REVIEW

File: core/cache/pom.xml [hunk 1]
```diff
@@ -3,7 +3,7 @@
     <parent>
         <artifactId>core</artifactId>
         <groupId>uk.gov.gchq.gaffer</groupId>
-        <version>1.13.5-SNAPSHOT</version>
+        <version>1.15.1-SNAPSHOT</version>
```
Comment (GCHQDev404) [inline core/cache/pom.xml:6]: Using maven <dependencyManagement> there should be no reference to versions.

Comment (m29827) [reply core/cache/pom.xml:6]: I think this is OK, it's specifying the parent module version. As an aside the branches containing v2 work have drifted behind develop which is now at 1.16.+ so there is some work to pull in the latest changes and bump the version on these branches eventually to 2.0.0-SNAPSHOT.

Comment (GCHQDev404) [reply core/cache/pom.xml:6]: OHHHHHH god! that's embarrassing.
Yes, this is correct. I thought I was looking at different part of the pom.

**Comment by r32575** [comment]
The conversion of the `rest-api` modules to use `spring-rest` rather than creating a `WAR`  has been put on hold.

**Pull request closed without merging**